examples/TemplateModule/TemplateModule.psd1

```diff
@@ -0,0 +1,59 @@
+﻿#
+# Module manifest for module 'TemplateModule'
+#
+
+@{
+
+# Version number of this module.
+ModuleVersion = '1.0'
+
+# ID used to uniquely identify this module
+GUID = '42225364-811c-47de-8828-54f79d0aa599'
+
+# Author of this module
+Author = 'Microsoft Corporation'
+
+# Company or vendor of this module
+CompanyName = 'Microsoft Corporation'
+
+# Copyright statement for this module
+Copyright = '(c) 2016 Microsoft Corporation. All rights reserved.'
+
+# Description of the functionality provided by this module
+Description = 'Provides an example module which contains a Plaster template'
+
+# Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
+FunctionsToExport = @()
+
+# Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
+CmdletsToExport = @()
+
+# Variables to export from this module
+VariablesToExport = '*'
+
+# Aliases to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no aliases to export.
+AliasesToExport = @()
+
+# List of all files packaged with this module
+# FileList = @()
+
+# Private data to pass to the module specified in RootModule/ModuleToProcess. This may also contain a PSData hashtable with additional module metadata used by PowerShell.
+PrivateData = @{
+
+    PSData = @{
+
+        Extensions = @(
+            @{
+                Module = "Plaster"

```

Comment:
I think I'll stick with what I'm currently doing just to get this first release out the door and then I'll try to solicit feedback from the community and the PowerShell team to see what improvements we can make.

I think it will be OK to have a deprecated format just for Plaster (for a little while anyway) if feedback encourages us to change it.  If we can get some agreement on it within the next month or so then I think not enough people will have shipped Plaster templates for it to matter :)